processor.py

```diff
@@ -18,9 +19,22 @@ def process(
             split_respect_sentence_boundary: Optional[bool] = None,
             id_hash_keys: Optional[List[str]] = None,
     ) -> List[Document]:
-        # print(documents)
+
+        mapped_values = OrderedDict()
+        mapped_values['merchant_name'] = "This business "
+        mapped_values['title'] = "sells the product "
+        mapped_values['brand'] = "of brand "
+        mapped_values['regprice'] = "the price is "
+        mapped_values['cat_mapping'] = "the item has been categorized as "
+        mapped_values['url'] = "the product can be found in the url "
+        mapped_values['descr'] = "the url describes the product as "
+
         c = ""
-        for i, j in documents.meta.items():
-            c += (f"{i} {j}. ")
+        for key, values in mapped_values.items():
+            if key in documents.meta:
+                if 'price' in key:
+                    c += f"{mapped_values[key]} {int(documents.meta[key]) / 100}. "
+                else:
+                    c += f"{mapped_values[key]} {documents.meta[key]}. "
         documents.content += ' ' + c
         return super().process(documents, id_hash_keys=id_hash_keys)
```

Comment:
you should add the params clean_whitespace, clean_header_footer.
at the beginning, the method has been created just to wrap and change the content.